README.md

```diff
@@ -14,7 +14,14 @@ Adds [nix](https://nixos.org/) language support for VSCode Editor.
 
   ![](./images/docs/md-embed-nix.png)
 
-* Full editing support with [rnix-LSP](https://github.com/nix-community/rnix-lsp)
+* Full editing support with any nix [LSP](https://microsoft.github.io/language-server-protocol/) implementation such as [rnix-lsp](https://github.com/nix-community/rnix-lsp) or [nil](https://github.com/oxalica/nil).
```

Comment:
Instead of saying any LSP with confidence, I think we can add some `generally` or such words? 

and mention that it has been tested to work with the following LSP servers ?

Reply:
Yeah, good point. I've updated the wording; let me know if that hits the spot or not.

Reply:
This looks good. Just a small nit though, please check.

Reply:
Works for me :+1: suggested change applied.